.travis.yml

```diff
@@ -18,12 +17,12 @@ matrix:
   include:
     - php: 5.3
       env: COMPOSER_FLAGS="--prefer-lowest"
-    - php: 5.4
-      env: COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
Only reason I did that is this http://php.net/eol.php
And looking at many bundles 5.3 and 5.4 is removed from travis. I'll revert to its original state.